### PR TITLE
Add PDF release GitHub action

### PR DESCRIPTION
**Pull request by aireilly**
Related to work done in https://github.com/redhat-documentation/supplementary-style-guide/pull/340, this PR adds a basic PDF output GitHub action to the SSG project.

Every merged commit triggers the PDF to be built as a release deliverable. Example: https://github.com/aireilly/adoc-test/releases/tag/v13

### CODE REVIEW

**Comment by aireilly** [comment]
Example SSG PDF build is here: https://drive.google.com/file/d/151vi4-fENMGmG2NuTq_loW0mmG_HSa-t/view?usp=sharing

**Comment by aireilly** [comment]
@bburt-rh Added date mechanism: 
image

...and yyyy-mm-dd date on the cover
image

**Comment by bergerhoffer** [comment]
@aireilly So this would generate a PDF every time the PR is updated? Where would the PDF be stored/available at? Is the main purpose for reviewing PRs?

Also, there's a merge conflict on the .gitignore if you can resolve that whenever you get a chance

**Comment by aireilly** [comment]
@bergerhoffer no this would only run when a PR is merged and main is updated. The PDF gets published in the releases section of the GitHub repo.

**Comment by bergerhoffer** [comment]
@aireilly Gotcha thanks. I don't have any reason to object to this, but just curious if we had someone asking for a PDF version?

**Comment by mportman12** [comment]
@bergerhoffer We did get a request for a PDF version in the SSG survey, although we decided not to prioritize that since folks can easily generate a PDF using the Print menu option. It turns out Aidan was independently working on this, though.

**Comment by aireilly** [comment]
I needed a PDF output for something else figured it would be useful here too. Does the print PDF option preserve links and xrefs? If it does maybe we don't need this PDF build at all.

**Comment by mportman12** [comment]
Regular links and anchors are preserved, at least in the PDF I created from a Chrome browser. There was some ugliness in the header and footer, which blocked off a couple of lines of text and had a weird shadow effect. Otherwise, it was very functional.

**Comment by aireilly** [comment]
@mportman12 OK I'll leave it to you folks to decide if you want this or not ¯\(ツ)/¯

**Comment by mportman12** [comment]
The concern before was really about whether it was worth it to put in the effort required to create generated PDFs. But since you already put in that effort, there's certainly no harm.

**Comment by joaedwar** [comment]
@mportman12 @bergerhoffer. Is this something that can be included in the style guide?

**Review by jseseCCS — APPROVED** [review]
lgtm

**Review by mportman12 — DISMISSED** [review]
LGTM

**Comment by mportman12** [comment]
@aireilly Can you resolve the merge conflict?

**Review by ndeevy — DISMISSED** [review]

**Comment by aireilly** [comment]
This is ready for merge pending review of one final addition. I added a link to the PDF in the supplementary_style_guide/introduction/about-guide.adoc section. The link will only become valid once the PR is merged and the workflow runs for the first time. @bergerhoffer @mportman12 @joaedwar PTAL

**Review by joaedwar — APPROVED** [review]
lgtm

**Pull request merged**
